__tests__/app.test.js

```diff
@@ -80,17 +76,64 @@ describe("GET /api/articles/:article_id", () => {
       .get("/api/articles/2")
       .expect(200)
       .then((response) => {
-        const {body} = response
-        const {article} = body
+        const { body } = response;
+        const { article } = body;
+        expect(article).toMatchObject({
+          author: expect.any(String),
+          title: expect.any(String),
+          article_id: expect.any(Number),
+          topic: expect.any(String),
+          created_at: expect.any(String),
+          votes: expect.any(Number),
+          article_img_url: expect.any(String),
+        });
+      });
+  });
+});
+
+describe("GET /api/articles", () => {
+  test("200: Responds with an array of all the articles.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        articles.forEach((article) => {
           expect(article).toMatchObject({
             author: expect.any(String),
             title: expect.any(String),
             article_id: expect.any(Number),
             topic: expect.any(String),
             created_at: expect.any(String),
             votes: expect.any(Number),
-            article_img_url: expect.any(String)
+            article_img_url: expect.any(String),
+            comment_count: expect.any(Number),
           });
-      })
+        });
+      });
+  });
+
+  test("It should NOT contain a body property.", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body }) => {
+        const { articles } = body;
+        articles.forEach((article) => {
+          expect(article).not.toHaveProperty("body");
+        });
+      });
```

Comment:
Nice! You could maybe include this in the above test but nothing wrong with it being a separate test either!